### ISSUE TITLE
Add fill pattern plugin 

### ISSUE THREAD

**Issue by ReichYang** [thedirtyfew/dash-leaflet#112]
Hi. I'm wondering if it is possible to add the pattern fills function when creating choropleth?

https://observablehq.com/@clhenrick/leaflet-pattern-fills

It looks like it needs the plugin https://github.com/teastman/Leaflet.pattern

**Comment by araichev** [comment]
@emilhe my company would be willing to help fund the implementation of this feature.
Would that help get it into the next release?

**Comment by emilhe** [comment]
@araichev Yes,it definitely would 👍. Do you have any particular deadline?

**Comment by araichev** [comment]
@emilhe 14 July (New Zealand time) would be good.
Is that feasible, and if so, at what cost?

**Comment by emilhe** [comment]
@araichev Yes, I believe that should be possible. I have sent you an email to discuss the details.

**Comment by joshred83** [comment]
Was there progress on this? I'm looking for the ability to generate shaded choropleths.

**Comment by araichev** [comment]
I found a workaround and didn't need the shading in the end, but i think @emilhe did progress the feature some.
